source/Halibut.Tests/ParallelRequestsFixture.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Halibut.Tests
 {
+    [Parallelizable(ParallelScope.Children)]
```

Comment:
Only let these tests run in parallel as they are resource intensive. If they still flake then we may need to change to NonParallel